portfolio/src/main/webapp/script.js

```diff
@@ -42,6 +42,13 @@ function getjson() {
         console.log(data[i].text);
     }
   });
+
+function getLoginStatus() {
+  fetch('/loginstatus').then(response => response.text()).then((loginstatus) => {
+    document.getElementById('loginstatus-container').innerText = loginstatus;
+  });
+}
```

Comment:
I think you wanted these lines to be placed after line 52...? I believe you figured this out over our chat, so just mentioning.